### DISCUSSION TITLE
Correct config file for multiple presets

### DISCUSSION THREAD

**Discussion by KainanMaki** [GerryDazoo/Slinger#456] [Q&A] [answered]
Thanks Gerry and everyone who contributed to Slinger!

I'm setting one slingbox 500 and I want to be sure about the right way of using the config file, to be able to have several bandwidth presets. Is it correct to consider every preset just as a different slingbox, sharing the same IP/password/etc?:

[SLINGBOXES]
sb1=500HD
sb2=500SD
sb3=500LD

[500HD]
sbtype="350/500/M1"
password=hidden
;finderid=hidden
;ipaddress=192.168.117.110
;port=5201
Resolution=16
FrameRate=60
VideoBandwidth=8000
VideoSmoothness=63
VideoSource=2
maxstreams=3
;StartChannel=155
;RemoteLock=yes
Remote=remote.txt

[500SD]
sbtype="350/500/M1"
password=hidden 
;ipaddress=192.168.117.101
;finderid=hidden
;port=5001
Resolution=12
FrameRate=30
VideoBandwidth=4000
VideoSmoothness=63
VideoSource=2
;RemoteLock=yes
;StartChannel=155
Remote=remote.txt

[500LD]
sbtype="350/500/M1"
password=hidden
;finderid=hidden
;ipaddress=192.168.117.129
;port=5301
Resolution=5
FrameRate=30
VideoBandwidth=2000
VideoSmoothness=63
VideoSource=2
Remote=remote.txt

**Comment by GerryDazoo** [comment, marked as answer]
Yes, they all should have the same IP address, port number and password.

**Reply by KainanMaki** [reply]
Thanks Gerry, it's working like a charm!

**Reply by socelillyhammer** [reply]
Hi Im a bit confused as I too am trying to now setup 2 different slingboxes on the same server.

each slingbox 500 I have has a different password and automatically connect to a different ports 5201 and 5208.  My setup is below.  I also learned that I have to keep the server and remote information seperate and must be used for both otherwise the server doesnt load.

Can somoene please advise?  Thank you

[SLINGBOX]
sb1=500Basement
sb2=500Master

[500Basement]
sbtype="350/500/M1/M2"
password=pw for this sling
ipaddress=192.168.1.182 [internal ip]
port=5208
finderid=51d65ab
name=Basement
Resolution=16
FrameRate=30
VideoBandwidth=1984
VideoSmoothness=59
IframeRate=4
VideoSource = 2

[500Master]
sbtype="300/500/M1/M2""
password=admin [had to reset this box]
finderid=BF7BD8
ipaddress=192.168.1.64 [internal ip]
port=5201
name=Master
Resolution=12
FrameRate=30
VideoBandwidth=1984
VideoSmoothness=59
IframeRate=4
VideoSource=2

[SERVER]
port=8080
maxstreams=1
enableremote=yes

[REMOTE]
code=2
include=remote.txt

**Reply by socelillyhammer** [reply]
Ive also read many other setups and still have issues.

This is what Im getting on this config:
[SLINGBOX]
sb1=basement
sb2=master

[basement]
sbtype="350/500/M1"
password=WEFasdf$#
finderid=51d65abf
ipaddress=192.168.1.182
port=5208
Resolution=16
FrameRate=30
VideoBandwidth=1984
VideoSmoothness=59
VideoSource=2
maxstreams=1
code=2
include=remote.txt

[master]
sbtype="350/500/M1"
password=admin 
finderid=BF7B 
ipaddress=192.168.1.64
port=5201
Resolution=16
FrameRate=30
VideoBandwidth=1984
VideoSmoothness=59
VideoSource=2
maxstreams=1
code=2
include=remote.txt


[SERVER]
port=51820
maxstreams=4
enableremote=yes

![InkedScreenshot 2023-01-02 154605](https://user-images.githubusercontent.com/119097385/210277046-6364c3ce-5f84-4e46-b243-e3f07579ff76.jpg)

**Reply by GerryDazoo** [reply]
It's [SLINGBOXES} not [SLINGBOX]

**Reply by socelillyhammer** [reply]
Thank you.  SLINGBOX TO SLINGBOXES was my problem.

Now, if there is an easier way to call up the box instead of using the url of 192.168.1.xxx:[port]/slingbox/basement or (whatever the name) and just use a url of http://abc123.com/slingbox/basement that would be amazing -- im connecting internally and not remotely as im using a vpn tunnel.

Cant thank you enough for all the help on this

**Reply by socelillyhammer** [reply]
it seems as though even though I am trying to access it internally and not remote, I have to have the ip addresses, finderid, and ports enabled to access the boxes.  But I keep getting this error

![InkedScreenshot 2023-01-02 201501](https://user-images.githubusercontent.com/119097385/210289297-4dd00717-43b4-4c21-bacc-08ce918f07de.jpg)

**Reply by socelillyhammer** [reply]
fixed the second problem - had to turn off the finderid

**Reply by GerryDazoo** [reply]
If the URL is toooo long for you. Set URLBase to something shorter. slingbox is the default and or change your sb=name to something shorter.  i.e.  URLBase=sb and Bsmt
